portal-frontend/Dockerfile

```diff
@@ -13,6 +13,8 @@ RUN npm ci
 # Copy the source code to the /app directory
 COPY . .
 
+ENV NODE_OPTIONS="--max-old-space-size=2048"
```

Comment:
did you run out of heap size?

Reply:
yeah, had to give docker 10gb or Ram even with this set